framework/src/physics/DiffusionCG.C

```diff
@@ -61,10 +65,14 @@ DiffusionCG::addFEKernels()
     assignBlocks(params, _blocks);
 
     // Transfer the diffusivity parameter from the Physics to the kernel
+    // From parameters
     if (isParamValid("diffusivity_matprop"))
       params.set<MaterialPropertyName>("diffusivity") =
           getParam<MaterialPropertyName>("diffusivity_matprop");
     else if (isParamValid("diffusivity_functor"))
+      params.set<FunctionName>("function") = getParam<MooseFunctorName>("diffusivity_functor");
```

Comment:
Functors aren't always functions. How does this work?

Reply:
see above, there is an error if the name is not the name of a function in the problem